docs/topics/service-account-token-secrets.md

```diff
@@ -0,0 +1,21 @@
+# Service account token secrets
+
+A `kubernetes.io/service-account-token` type of Secret is used to store a token credential that identifies a ServiceAccount. This is a legacy mechanism that provides long-lived ServiceAccount credentials to Pods.
+
+You can mount service account tokens as Kubernetes Secrets in Pods. These tokens don't expire and don't rotate. This method is not recommended, especially at scale, because of the risks associated with static, long-lived credentials.
+
+## Autogeneration
+
+Previously, these Secret-based service account tokens were auto generated for ServiceAccounts. In Kubernetes v1.27 and later, Kubernetes forced the `LegacyServiceAccountTokenNoAutoGeneration`` feature gate to true, preventing Kubernetes from automatically creating these service account token secrets for ServiceAccounts.

```

Comment:
`LegacyServiceAccountTokenNoAutoGeneration``

extra "`"

Reply:
I would change format to "feature gate value to `true`"

Reply:
Done, thanks!